packages/dx-chart-core/src/plugins/zoom-pan/computeds.ts

```diff
@@ -153,3 +153,9 @@ export const isKeyPressed = (event: MouseEvent, key: string) => event[`${key}Key
 export const getWheelDelta = ({ wheelDelta, deltaY }: { wheelDelta?: number, deltaY?: number }) => {
   return wheelDelta !== undefined ? wheelDelta : deltaY! * -30; // deltaY for FF
 };
+
+/** @internal */
+export const getCoordsWithOffset = (e: any, offset: NumberArray): NumberArray => {
+  const { pageX, pageY } = e.touches ? e.touches[0] : e;
```

Comment:
Let's extract a function.
```jsx
const getEventCoords = (e: any): NumberArray => {
  const { pageX, pageY } = e.touches ? e.touches[0] : e;
  return [pageX, pageY];
};
```
That's because similar code is used in `EventTracker`.

And `getCoordsWithOffset` is also used in `EventTracker`.